crates/cache/src/metrics.rs

```diff
@@ -23,39 +23,39 @@ use opentelemetry::{
 
 static METER: LazyLock<Meter> = LazyLock::new(|| global::meter("results_cache"));
 
-pub(crate) static SIZE: LazyLock<Gauge<u64>> = LazyLock::new(|| {
+pub(crate) static SIZE_BYTES: LazyLock<Gauge<u64>> = LazyLock::new(|| {
     METER
-        .u64_gauge("results_cache_size")
+        .u64_gauge("results_cache_size_bytes")
         .with_description("Size of the cache in bytes.")
-        .with_unit("b")
+        .with_unit("By")
         .init()
 });
 
-pub(crate) static MAX_SIZE: LazyLock<Gauge<u64>> = LazyLock::new(|| {
+pub(crate) static MAX_SIZE_BYTES: LazyLock<Gauge<u64>> = LazyLock::new(|| {
     METER
-        .u64_gauge("results_cache_max_size")
+        .u64_gauge("results_cache_max_size_bytes")
         .with_description("Maximum allowed size of the cache in bytes.")
-        .with_unit("b")
+        .with_unit("By")
         .init()
 });
 
-pub(crate) static REQUEST_COUNT: LazyLock<Counter<u64>> = LazyLock::new(|| {
+pub(crate) static REQUESTS: LazyLock<Counter<u64>> = LazyLock::new(|| {
     METER
-        .u64_counter("results_cache_request_count")
+        .u64_counter("results_cache_requests")
         .with_description("Number of requests to get a key from the cache.")
         .init()
 });
 
-pub(crate) static HIT_COUNT: LazyLock<Counter<u64>> = LazyLock::new(|| {
+pub(crate) static HITS: LazyLock<Counter<u64>> = LazyLock::new(|| {
     METER
-        .u64_counter("results_cache_hit_count")
-        .with_description("Cache hit count")
+        .u64_counter("results_cache_hits")
+        .with_description("Cache hit count.")
         .init()
 });
 
-pub(crate) static ITEM_COUNT: LazyLock<Counter<u64>> = LazyLock::new(|| {
+pub(crate) static ITEMS: LazyLock<Counter<u64>> = LazyLock::new(|| {
     METER
-        .u64_counter("results_cache_item_count")
-        .with_description("Number of items currently in the cache.")
+        .u64_counter("results_cache_items")
+        .with_description("Number of items in the cache.")
```

Comment:
If this is a counter, then this description should really be "Number of items ever in the cache"?

Reply:
But given the previously comment, it seems like it should be a gauge of the number of items currently in the cache, not a counter.

Reply:
Thank you - will change to gauge as it makes more sense

Reply:
Updated